### DISCUSSION TITLE
Run amd64 docker images using box64.

### DISCUSSION THREAD

**Discussion by Abdalnablse10** [ptitSeb/box64#307] [General]
Is it possible?

**Comment by ptitSeb** [comment]
probably not for now (libfuse is not even working yet)

**Comment by MRColorR** [comment]
Any update on this?

**Reply by ptitSeb** [reply]
well, libfuse is supported, and you can build a staticaly linked version of box64. It seems it's not yet working 100% but there have been numerous progress yes.

**Reply by ptitSeb** [reply]
there is a "static" build of box64 now. So it might work with this build.
